test/unittests/evm_other_test.cpp

```diff
@@ -56,3 +56,12 @@ TEST_F(evm_other, loop_full_of_jumpdests)
160200383360303811215156179ae5760006000fd5b8083019250508135905060208201915067ffffffffffffffff8111156179d45760006000fd5b6020810236038213156179e75760006000fd5b600381811015156179f457fe5b9050602002016020617a09919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617a3b5760509050618b07565b6004836003600481101515617a4c57fe5b6020028101803560016080038336030381121515617a6a5760006000fd5b808301925050506003600481101515617a7f57fe5b6020028101803560016020038336030381121515617a9d5760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617ac35760006000fd5b602081023603821315617ad65760006000fd5b9050141515617ae85760519050618b07565b7fa800000000000000000000000000000000000000000000000000000000000000836003600481101515617b1857fe5b6020028101803560016080038336030381121515617b365760006000fd5b808301925050506003600481101515617b4b57fe5b6020028101803560016020038336030381121515617b695760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617b8f5760006000fd5b602081023603821315617ba25760006000fd5b60008181101515617baf57fe5b9050602002016020617bc4919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617bf65760529050618b07565b7f2000000000000000000000000000000000000000000000000000000000000000836003600481101515617c2657fe5b6020028101803560016080038336030381121515617c445760006000fd5b808301925050506003600481101515617c5957fe5b6020028101803560016020038336030381121515617c775760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617c9d5760006000fd5b602081023603821315617cb05760006000fd5b60018181101515617cbd57fe5b9050602002016020617cd2919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617d045760539050618b07565b7fc700000000000000000000000000000000000000000000000000000000000000836003600481101515617d3457fe5b6020028101803560016080038336030381121515617d525760006000fd5b808301925050506003600481101515617d6757fe5b6020028101803560016020038336030381121515617d855760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617dab5760006000fd5b602081023603821315617dbe5760006000fd5b60028181101515617dcb57fe5b9050602002016020617de0919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617e125760549050618b07565b7f6c00000000000000000000000000000000000000000000000000000000000000836003600481101515617e4257fe5b6020028101803560016080038336030381121515617e605760006000fd5b808301925050506003600481101515617e7557fe5b6020028101803560016020038336030381121515617e935760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115617eb95760006000fd5b602081023603821315617ecc5760006000fd5b60038181101515617ed957fe5b9050602002016020617eee919081019061c026565b7effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916141515617f205760559050618b07565b600482506004141515617f365760569050618b07565b6004826000600481101515617f4757fe5b6020028101803560016020038336030381121515617f655760006000fd5b8083019250508135905060208201915
```

Comment:
Note to self: This is currently failing.

Reply:
The issue might be related to the code size. First of all this one is over 50k. evmone only supports code size up to 24k officially. I also tried to make it smaller, but the issue goes away if the code is trimmed by ~1k.

Reply:
I agree. Am curious why this isn't an issue in evmone 0.1 (ref https://github.com/ethereum/solidity/issues/7776#issuecomment-558136555)